### PR TITLE
Konnect Rename: Dev Portal section

### PR DESCRIPTION
**Pull request by Guaris**
This PR renames: 

```
Runtime Manager to Gateway Manager
Runtime Groups to Control Plane
Composite Runtime Groups to Control Plane Group
Runtime Instances to Gateway Instances
```

Also did some copy edits and minor restructuring. 

https://konghq.atlassian.net/browse/DOCU-3374

### CODE REVIEW

File: app/konnect/dev-portal/applications/application-overview.md
Comment (cloudjumpercat) [inline app/konnect/dev-portal/applications/application-overview.md:20]:
```suggestion
- View the status of an application registration to an API product.
```
I think, since devs register an app against an API product?

File: app/konnect/dev-portal/index.md
Comment (cloudjumpercat) [inline app/konnect/dev-portal/index.md:10]:
```suggestion
In {{site.konnect_short_name}}, API products can be published to the Dev Portal with one click. Published API products become immediately available to users who have access to the Dev Portal. When an API product is published, the API spec and any service documentation becomes discoverable. We use the term discoverable here because the Dev Portal can create a unified API experience where a developer can navigate through the different APIs that are available, read documentation, test endpoints within the Dev Portal, and register to create applications for specific APIs. 
```

**Review by cloudjumpercat — COMMENTED** [review]
Left a few comments, overall great revisions!

File: app/konnect/dev-portal/applications/dynamic-client-registration/azure.md
Comment (Guaris) [inline app/konnect/dev-portal/applications/dynamic-client-registration/azure.md:90]:
```suggestion
Where `example.com` is the address of the data plane node.
```

File: app/konnect/dev-portal/applications/enable-app-reg.md
Comment (cloudjumpercat) [inline app/konnect/dev-portal/applications/enable-app-reg.md:121]:
```suggestion
In the `default` control plane group, applications are linked to {{site.base_gateway}} consumers and use the `acl` plugin to control access between an application’s consumers and an API product version. For all other control planes, applications are not linked to {{site.base_gateway}} consumers.
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Left one small comment, so I'll let you merge this one (also wasn't sure if you wanted Lena to review this as well).

**Pull request merged**
